defaults/main.yml

```diff
@@ -11,13 +11,13 @@ riak_usr_lib:       /usr/lib
 
 riak_admin:   '/usr/sbin/riak-admin'
 
-riak_node_name:     'riak@127.0.0.1'
+riak_node_name:     'riak@{{ ansible_fqdn }}'
```

Comment:
Nice, I don't think I was aware of this hostvar and was trying to think of a better way to handle this.